portal/models/user.py

```diff
@@ -1084,8 +1090,8 @@ def allow_org_change(org, user, acting_user):
 
             """
             if (not acting_user.has_role(ROLE.ADMIN.value)
-                and (acting_user.has_role(ROLE.STAFF.value)
-                     or acting_user.has_role(ROLE.STAFF_ADMIN.value))
+                    and (acting_user.has_role(ROLE.STAFF.value)
```

Comment:
Too much indentation

Reply:
@ivan-c the double indent is so that it doesn't have the same indent as line 1096. Should I revert?